Gemfile

```diff
@@ -23,4 +23,8 @@ group(:dev) do
   gem 'guard'
   gem 'guard-rspec'
   gem 'ruby_gntp'
+  gem 'chefspec'
+  gem 'rubocop'
+  gem 'knife-spork'
+  gem 'kitchen-vagrant'
```

Comment:
Can you remind me why this is? I have a vague recollection of this, but don't remember the details.